bloom/generators/debian/templates/ament_python/copyright.em

```diff
@@ -1 +1,11 @@
-@(Copyright)
\ No newline at end of file
+Format: Bloom subset of https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: @(Name)
+@[if BugTracker and BugTracker != '']Upstream-Contact: @(BugTracker)@\n@[end if]@
```

Comment:
This could optionally be simplified since empty strings in Python coerce to boolean as `False`. (this is assuming empy conditionals behave like Python ones)

This same suggestion applies to the other copyright files but I wouldn't consider it a blocker, especially if you like the explicitness of the current form.

```suggestion
@[if BugTracker]Upstream-Contact: @(BugTracker)@\n@[end if]@
```

Reply:
My use of this pattern in Bloom dates back to the original RPM generator, where an empty "website" URL was breaking RPM builds. It's been ~8 years since I did that, and I don't remember the exact circumstances, but it was evidently necessary to explicitly check at that time. I just checked and it doesn't appear to be necessary on my machine (using empy 3.3.4 and python 3.9.7).

Maybe empy has changed. I'll drop the explicit comparison.